### ISSUE TITLE
Änderung einer Wiedervorlage / Frist

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#207]
Ein gutes neues Jahr wünsche ich. Die Bearbeitung (Rechte Maustaste) einer Wiedervorlage ist ja jetzt möglich. Aber es ist nicht möglich, bei der Bearbeitung die Wiedervorlage zur Frist oder umgekehrt zu machen. Da muss man alles löschen und neu anfangen.....

**Comment by j-dimension** [comment]
Wie häufig kommt so ein Fall denn vor? Findet die Entscheidung "Frist oder WV" nicht bewusst bei Anlage statt?

**Comment by Mucki526** [comment]
Richtig, die Anlage einer falschen Frist oder WV kommt nicht so häufig vor. Aber wenn ich die Möglichkeit des Bearbeitens habe, weil mir ein Fehler passiert ist, ist es ja nur konsequent / Konsistent, wenn ich auch die Art des Eintrags (Frist/Widervorlage) ändern kann und nicht nur das Datum und die Anmerkung. Ich schreibe halt hier, wenn ich einen Fehler bemerke oder / und  im "workflow" an einer Stelle deshalb mehr als zweimal ins Stocken komme.... Natürlich geht es auch "ohne" die Funktion, es geht ja immer irgendwie ohne eine bestimmte Funktion, es kostet halt oft aber mehr Zeit - jedesmal wenn man sie brauchen könnte.

**Comment by j-dimension** [comment]
Danke für die Erläuterung. Das war nicht als Kritik gemeint :-)  Ich versuche nur den Arbeitsfluss möglichst genau zu verstehen, und manchmal auch Interessen auszugleichen (bspw. wenn es Meinungen gibt wie "aber eine Frist zur WV machen zu können ist gefährlich!").

**Comment by Mucki526** [comment]
Ist mir doch klar :-).  Darum poste ich ja auch immer gleich, wenn mir im workflow was "unterkommt", selbst wenn mal eine Doublette dabei ist. Eine Frist durch Änderung zur Wiedervorlage zu machen ist nicht weniger gefährlich als sie zu löschen.... :-).
